src/chain.h

```diff
@@ -429,10 +430,11 @@ class CBlockIndex
 
     std::string ToString() const
     {
-        return strprintf("CBlockIndex(pprev=%p, nHeight=%d, merkle=%s, hashBlock=%s)",
+        return strprintf("CBlockIndex(pprev=%p, nHeight=%d, merkle=%s, hashBlock=%s, nSolution=%s)",
```

Comment:
Nit: we weren't printing the solution previously here because it is very large, and not actually useful outside the header (vs the other values that are useful in the context of other block-related debug output).

Reply:
What do you think about changing this to just print the result of `HasSolution()`?

Reply:
Yeah, that works fine.